src/System.Drawing.Common/src/System/Drawing/Drawing2D/HatchBrush.cs

```diff
@@ -2,137 +2,69 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+
 namespace System.Drawing.Drawing2D
 {
-    using System.Runtime.InteropServices;
-    using System.Diagnostics;
-
-    /**
-     * Represent a HatchBrush brush object
-     */
-    /// <include file='doc\HatchBrush.uex' path='docs/doc[@for="HatchBrush"]/*' />
-    /// <devdoc>
-    ///    Defines a rectangular brush with a hatch
-    ///    style, a foreground color, and a background color.
-    /// </devdoc>
     public sealed class HatchBrush : Brush
     {
-        /**
-         * Create a new hatch brush object
-         */
-        /// <include file='doc\HatchBrush.uex' path='docs/doc[@for="HatchBrush.HatchBrush"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Drawing.Drawing2D.HatchBrush'/> class with the specified <see cref='System.Drawing.Drawing2D.HatchStyle'/> and foreground color.
-        ///    </para>
-        /// </devdoc>
-        public HatchBrush(HatchStyle hatchstyle, Color foreColor) :
-            this(hatchstyle, foreColor, Color.FromArgb(unchecked((int)0xff000000)))
+        public HatchBrush(HatchStyle hatchstyle, Color foreColor) : this(hatchstyle, foreColor, Color.FromArgb(unchecked((int)0xff000000)))
         {
         }
 
-        /// <include file='doc\HatchBrush.uex' path='docs/doc[@for="HatchBrush.HatchBrush1"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.Drawing.Drawing2D.HatchBrush'/> class with the specified <see cref='System.Drawing.Drawing2D.HatchStyle'/>,
-        ///       foreground color, and background color.
-        ///    </para>
-        /// </devdoc>
         public HatchBrush(HatchStyle hatchstyle, Color foreColor, Color backColor)
-        {
-            IntPtr brush = IntPtr.Zero;
-            int status = SafeNativeMethods.Gdip.GdipCreateHatchBrush(unchecked((int)hatchstyle), foreColor.ToArgb(), backColor.ToArgb(), out brush);
-
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
+        { 
+            int status = SafeNativeMethods.Gdip.GdipCreateHatchBrush(unchecked((int)hatchstyle), foreColor.ToArgb(), backColor.ToArgb(), out IntPtr brush);
```

Comment:
In cases like this, I'd actually prefer it if we didn't use the new C# syntax and left the variable defined on its own line.  Otherwise, it's too difficult when reading the code to see where brush is actually declared.